.github/workflows/version_handling.yml

```
@@ -32,7 +32,12 @@ jobs:
         run: |
           [[ "$(~/.local/bin/subctl version)" =~ ^"subctl version: v"[0-9.]+-rc[0-9]+$ ]]
 
-      - name: Check we got the right version (devel/release)
-        if: matrix.version == 'devel' || startsWith(matrix.version, 'release')
+      - name: Check we got the right version (release-0.8)
+        if: matrix.version == 'release-0.8'
         run: |
           [[ "$(~/.local/bin/subctl version)" =~ ^"subctl version: subctl-${{ matrix.version }}"(-[0-9]+-g[a-f0-9]+)?$ ]]
+
+      - name: Check we got the right version (devel/release > 0.8)
+        if: matrix.version == 'devel' || (startsWith(matrix.version, 'release') && matrix.version != 'release-0.8')
+        run: |
+          [[ "$(~/.local/bin/subctl version)" =~ ^"subctl version: ${{ matrix.version }}" ]]
```

We can simplify this by just checking that the output contains the version number, ie using regex pattern "*${{ matrix.version }}*" or whatever the correct syntax is.

```suggestion
      - name: Check we got the right version
        run: |
          [[ "$(~/.local/bin/subctl version)" =~ *${{ matrix.version }}*]]
```